identity/src/main/java/com/stripe/android/identity/navigation/BaseErrorFragment.kt

```diff
@@ -0,0 +1,38 @@
+package com.stripe.android.identity.navigation
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import com.google.android.material.button.MaterialButton
+import com.stripe.android.identity.databinding.BaseErrorFragmentBinding
+
+/**
+ * Base error fragment displaying error messages and two buttons
+ */
+internal abstract class BaseErrorFragment : Fragment() {
+    protected lateinit var title: TextView
+    protected lateinit var message1: TextView
+    protected lateinit var message2: TextView
+    protected lateinit var topButton: MaterialButton
+    protected lateinit var bottomButton: MaterialButton
```

Comment:
if any of these are not set, it will be filled with placeholder text, right? Rather than just exposing these, would it be better to have an abstract setup method called from the constructor with values for each of these?

Reply:
hmm I thought of that options too, but in some cases the topButton or message2 field will be hidden, exposing a "setText" or "shouldHide" for each of the field seems too verbose, so I ended up just exposing a onCustomizingViews() before `onCreateView` returns for a more flexible override